coalib/misc/Exceptions.py

```diff
@@ -3,6 +3,8 @@
 from coalib.misc import Constants
 from coalib.output.printers.LogPrinter import LogPrinter
 
+from pkg_resources import ContextualVersionConflict, VersionConflict
```

Comment:
See https://github.com/jaraco/setuptools/blob/master/pkg_resources/__init__.py#L359

Still go ahead? It might fail for some people.